### PR TITLE
Make buttons disabled during process work

### PR DESCRIPTION
**Pull request by montlikadani**
This changes all buttons to disabled or enabled depending on whether the process is started or completed and prevents the user from performing unnecessary click actions.

### CODE REVIEW

File: src/main/java/com/shanebeestudios/mcdeop/Processor.java [hunk 1]
```diff
@@ -75,6 +79,7 @@ public void init() {
             long finish = Duration.ofMillis(System.currentTimeMillis() - start).toMinutes();
             Logger.info("Process finished in %s minutes!", finish);
             if (app != null) {
+                app.toggleControls();
```
Comment (gekkedev) [inline src/main/java/com/shanebeestudios/mcdeop/Processor.java:82]: This seems alright if compiling is successful, but where are you handling failure?

Comment (montlikadani) [reply src/main/java/com/shanebeestudios/mcdeop/Processor.java:82]: You're right, I accidentally managed to ignore this if there is a runtime exception.

**Review by gekkedev — APPROVED** [review]
downloaded, tested, button is now locked during compilation :-)

**Review by ShaneBeee — APPROVED** [review]
As long as it works, it looks good to me 👍

**Pull request merged**